exercises/00-init/cypress/plugins/index.js

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
```

Comment:
we seem to have many boilerplate files there. do we need them? (will be need them?)

Reply:
Removed, see 9b6e012